### PR TITLE
VI-44_new

### PR DESCRIPTION
**Pull request by dkruczek**

### CODE REVIEW

**Comment by dkruczek** [comment]
@astyczen

File: my-project/public/css/show.css [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+{
+    float: left;
+}
+
+.col-md-8_custom
+{
+    float: left;
+}
+
+.MainImage
```
Comment (astyczen) [inline my-project/public/css/show.css]: wrong class name

File: my-project/src/Controller/firstPage.php [hunk 2]
```diff
@@ -22,7 +22,7 @@ public function date()
     {
         $today=date('d-m-Y');
 
-        return new Response('<html><body> <h1>Witaj na stronie</h1><p>Dziś jest: '.$today.'</p> <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum interdum imperdiet diam, a fermentum diam consectetur at. Fusce quis mauris eu ex fringilla fringilla. Sed nec sapien et nibh vehicula placerat. Fusce condimentum porta dictum. Sed ac est cursus, elementum velit ac, tempor lectus. Curabitur lobortis est et nulla elementum, vitae congue lectus condimentum. Vivamus feugiat, velit id tristique semper, urna diam lacinia sapien, et molestie eros turpis quis risus. Phasellus gravida ante odio, at ullamcorper nunc sodales at. Vivamus venenatis varius erat ac hendrerit. Suspendisse leo purus, rutrum sed nulla id, varius cursus orci.</p></body></html>'
```
Comment (astyczen) [inline my-project/src/Controller/firstPage.php:25]: still the unneded change

File: my-project/src/Form/ImageType.php [hunk 3]
```diff
@@ -21,15 +21,15 @@ public function buildForm(FormBuilderInterface $builder, array $options)
     {
         $builder
             ->add('path', FileType::class, [
-                'label' => 'Add Image',
+                'label'     => 'Add Image',
```
Comment (astyczen) [inline my-project/src/Form/ImageType.php]: tabs not needed

File: my-project/src/Form/ProductType.php [hunk 4]
```diff
@@ -18,15 +18,21 @@ public function buildForm(FormBuilderInterface $builder, array $options)
...
             ->add('category',EntityType::class, array(
                 'class'=>ProductCategory::class,
                 'choice_label'=>function($category)
                 {
                     return $category->getName();
-                }
+                }))
+            ->add('picture',FileType::class, array('label'=>'Add Main Image','data_class'=>null))
+            ->add('images',CollectionType::class, array(
+                'entry_type'        =>ImageTypeProduct::class,
```
Comment (astyczen) [inline my-project/src/Form/ProductType.php:29]: allign all arrows

File: my-project/templates/base.html.twig [hunk 5]
```diff
@@ -16,14 +16,13 @@
...
             <link rel="stylesheet" href="{{ asset('css/Shared/coreStyle.css') }}">
         {% endblock %}
     </head>
-    <body>
-        {% include 'menu/menu.html.twig' %}
-        <div id="base" class="container">
-            {% block body %}{% endblock %}
-        </div>
-        {% block javascripts %}{% endblock %}
+    <body id="base" class="container-fluid">
```
Comment (astyczen) [inline my-project/templates/base.html.twig]: format html to have proper cuts in

**Review by astyczen — COMMENTED** [review]
+/**
+ * VI-44 - Add and Edit gallery
+ *
+ * @category   JS
+ * @package    Virtua_ProductGallery
+ * @copyright  Copyright (c) Virtua
+ * @author     Dawid Kruczek
+ */

**Review by astyczen — APPROVED** [review]

**Pull request merged**